workinggroups/README.md

```diff
@@ -18,3 +24,21 @@ At a minimum, please include this information:
 * The location of meetings / agenda / notes
 * Initial interested parties to show that there are multiple people across multiple orgs interested
 * The chair(s) and TOC sponsor being explicitly listed so they are discoverable
+
+## Working Groups (as part of a TAG)
+
+This section outlines the process for working groups
+that are part of a TAG.
+
+### Creating a new WG
+
+To request for a new working group as part of a TAG, an issue must be created
+against the `cncf/toc` repo stating the charter of the WG.
+
+The issue must be approved by:
+
+- TOC liaisons for the respective TAG.
+    - If a TAG has 3 TOC liaisons, the issue must be approved by at least 2 liaisons.
+- At least one of the TAG leads or any process spelled out in a TAG's charter.
```

Comment:
One of the TAG chairs/leads?

Reply:
> One of the TAG chairs/leads?

@cathyhongzhang leads should cover chairs.

Scoping to requiring at least one to account for availabilty of chairs, vacations, etc.